### PR TITLE
Fix lost encoding for multi meta-model. Close #81 (new PR)

### LINKED ISSUE #81: Encoding is lost when using Multi meta-model
I have model (meta-model A) which imports models of another meta-model B. Model-Files of model B do not work with `utf-8` and used encoding is lost.

I assumed the encoding would be propagated.

```
Traceback (most recent call last):
  File "...\Generator.py", line 36, in getModel
    mmA.model_from_file(filePath, encoding='iso-8859-1')
  File "...\Python\Python37\lib\site-packages\textx-1.7.1-py3.7.egg\textx\metamodel.py", line 505, in model_from_file
    return self.internal_model_from_file(file_name, encoding, debug)
  File "...\Python\Python37\lib\site-packages\textx-1.7.1-py3.7.egg\textx\metamodel.py", line 542, in internal_model_from_file
    is_main_model=is_main_model)
  File "...\Python\Python37\lib\site-packages\textx-1.7.1-py3.7.egg\textx\model.py", line 260, in get_model_from_file
    is_main_model=is_main_model)
  File "...\Python\Python37\lib\site-packages\textx-1.7.1-py3.7.egg\textx\model.py", line 285, in get_model_from_str
    is_main_model=is_main_model)
  File "...\Python\Python37\lib\site-packages\textx-1.7.1-py3.7.egg\textx\model.py", line 614, in parse_tree_to_objgraph
    scope_provider.load_models(model)
  File "...\Python\Python37\lib\site-packages\textx-1.7.1-py3.7.egg\textx\scoping\providers.py", line 297, in load_models
    self._load_referenced_models(model)
  File "...\Python\Python37\lib\site-packages\textx-1.7.1-py3.7.egg\textx\scoping\providers.py", line 282, in _load_referenced_models
    filename_pattern, model=model, glob_args=self.glob_args)
  File "...\Python\Python37\lib\site-packages\textx-1.7.1-py3.7.egg\textx\scoping\__init__.py", line 159, in load_models_using_filepattern
    self.load_model(the_metamodel, filename, is_main_model))
  File "...\Python\Python37\lib\site-packages\textx-1.7.1-py3.7.egg\textx\scoping\__init__.py", line 213, in load_model
    is_main_model=is_main_model)
  File "...\Python\Python37\lib\site-packages\textx-1.7.1-py3.7.egg\textx\metamodel.py", line 542, in internal_model_from_file
    is_main_model=is_main_model)
  File "...\Python\Python37\lib\site-packages\textx-1.7.1-py3.7.egg\textx\model.py", line 255, in get_model_from_file
    model_str = f.read()
  File "...\Python\Python37\lib\codecs.py", line 708, in read
    return self.reader.read(size)
  File "...\Python\Python37\lib\codecs.py", line 504, in read
    newchars, decodedbytes = self.decode(data, self.errors)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x96 in position 4121: invalid start byte
```

### PR DESCRIPTION
**Pull request by goto40**
replacement for PR #82.

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```
Comment (igordejanovic) [inline .idea/vcs.xml:6]: @goto40 Please remove this file. We shouldn't keep IDE artifacts in the project repo.

**Comment by goto40** [comment]
I am sorry. You are completely right. I already merged the branch and
removed the file from the master just now.

Am So., 2. Sep. 2018 um 21:29 Uhr schrieb Igor Dejanović <
notifications@github.com>:

**Comment by igordejanovic** [comment]
No problem :) BTW, we should update the docs to include encoding support before the next release.

**Comment by goto40** [comment]
Ok.

Am So., 2. Sep. 2018 um 21:58 Uhr schrieb Igor Dejanović <
notifications@github.com>:

**Pull request merged**